### PR TITLE
Do not allow URLs to leak outside of the app scope

### PR DESCRIPTION
**Pull request by alancutter**
This updates the focus-existing behaviour to check first whether the |application context| is even in the app scope before not navigating it. This can happen if the user navigates outside of the app scope in an existing app window prior to launching the web app again while `focus-existing` is active.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -358,10 +358,15 @@ <h2>
...
-                      resource|.
+                      [=applied=]:
+                    <ol>
+                      <li>
+                        Return the result of running the steps to [=create a new
+                        application context=] passing |manifest|,
+                        |launch params|.{{LaunchParams/targetURL}} and |POST
+                        resource|.
+                      </li>
+                    </ol>
```
Comment (alancutter) [inline index.html:369]: Is is an unrelated style change to indent the if body.

**Review by ewilligers — APPROVED** [review]
